### ISSUE TITLE
Launch Protein Viewer from File failed

### ISSUE THREAD

**Issue by zirui** [a-r-j/VSCoding-Sequence#10]
"Start Protein Viewer' with PDB works, 
but when select 'Launch Protein Viewer from Files(s)', vscode throw error msg:
"Command 'Launch Protein Viewer from File(s)' resulted in an error (Cannot read properties of undefined (reading 'map'))

vscode version: 
```
Version: 1.67.1 (Universal)
Commit: da15b6fd3ef856477bf6f4fb29ba1b7af717770d
Date: 2022-05-06T12:37:16.526Z (4 days ago)
Electron: 17.4.1
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin x64 20.6.0
```

**Comment by a-r-j** [comment]
Hmm. I'm struggling to reproduce this @zirui. What sort of files are you trying to open? Maybe you can share them?

**Comment by zirui** [comment]
> 
i'm trying to open a pdb file(download from rcsb.org),
but i think this error was not caused by the file itself, because it occurs before open files(when select 'Launch Protein Viewer from Files' in command palette)

**Comment by a-r-j** [comment]
Ah, gotcha. You shouldn’t use the command palette to open from files. If you right click on the file(s) in the file explorer & select ‘Launch Protein Viewer from Files’ it should do the trick. Check out the GIF in the readme for an example :)

**Comment by a-r-j** [comment]
@zirui How did you get on? Can I close the issue?

**Comment by zirui** [comment]
> 

right click on the file in the file explorer & select 'Launch Protein Viewer from Files' works!
but i think  it will be more convenient if view pdb from command palette  is supported

**Comment by a-r-j** [comment]
I agree that would be a nice feature. Unfortunately, I don’t think I’m going to have bandwidth to add it in the foreseeable future. Apologies.

Happy to support if you want to contribute a PR.

>
